tst/CTA.FeatureDetection.Tests/Fixtures/TestProjectsSetupFixture.cs

```diff
@@ -65,12 +68,14 @@ public class TestProjectsSetupFixture
         public static string CoreWCFServiceConfigProjectPath => Path.Combine(_targetDir, _wcfDir, _wcfConfigBasedProjectDir, "WCFConfigBasedProject.csproj");
         public static string CoreWCFServiceCodeProjectPath => Path.Combine(_targetDir, _wcfDir, _wcfCodeBasedProjectDir, "WCFCodeBasedProject.csproj");
         public static string WCFClientProjectPath => Path.Combine(_targetDir, _wcfDir, "WCFClient.csproj");
+        public static string WebFormsProjectPath => @"D:\TestRepos\TestProjects\net472\FeatureDetection\WebForms\ASP.NET-WebForms\ASP.NET-WebForms.csproj";
```

Comment:
Same as above, can we replace this string literal?

e.g. `WebFormsProjectPath => Path.Combine(_targetDir, _webFormsDir, "ASP.NET-WebForms", "WebForms.csproj");`